src/Classes/Providers/WindowsDISMParcelProvider.ps1

```diff
@@ -0,0 +1,57 @@
+class WindowsDISMParcelProvider : ParcelProvider
+{
+    WindowsDISMParcelProvider() : base('Windows DISM', $false, [string]::Empty) {}
+
+    [bool] TestProviderInstalled([hashtable]$_context)
+    {
+        if ($_context.os.type -ine 'windows') {
+            throw 'Windows DISM is only supported on Windows.....'
+        }
+
+        return $true
+    }
+
+    [string] GetPackageInstallScript([ParcelPackage]$_package)
+    {
+        return "Invoke-Expression -Command 'DISM /Online /Enable-Feature /All /FeatureName:$($_package.Name) /NoRestart'"
+    }
+
+    [string] GetPackageUninstallScript([ParcelPackage]$_package)
+    {
+        return "Invoke-Expression -Command 'DISM /online /Disable-feature /FeatureName:$($_package.Name)'"
+    }
+
+    [string] GetProviderAddSourceScript([string]$_name, [string]$_url)
+    {
+        return $null
+    }
+
+    [bool] TestPackageInstalled([ParcelPackage]$_package)
+    {
+        $checkDismPackage = Invoke-Expression -Command "dism /online /get-featureinfo /featurename:$($_package.Name )" -ErrorAction Stop
+        $checkDismPackageState = $checkDismPackage -imatch "State"
+        if ($checkDismPackageState -ilike "*Disabled*")
+        {return $false}
+        else 
+        {return $true}
```

Comment:
You can change all of this to just be:
```powershell
return ($checkDismPackageState -inotlike "*Disabled*")
```

Reply:
You could, but it wouldn't look like ASCII art like my code does.